src/components/application_manager/test/commands/mobile/perform_audio_pass_thru_test.cc

```diff
@@ -109,103 +127,461 @@ class PerformAudioPassThruRequestTest
   }
 
   void ResultCommandExpectations(MessageSharedPtr msg,
-                                 const std::string& info) {
+                                 const std::string& info,
+                                 const int32_t code,
+                                 const bool success) {
     EXPECT_EQ((*msg)[am::strings::msg_params][am::strings::success].asBool(),
-              true);
-    EXPECT_EQ(
-        (*msg)[am::strings::msg_params][am::strings::result_code].asInt(),
-        static_cast<int32_t>(hmi_apis::Common_Result::UNSUPPORTED_RESOURCE));
+              success);
+    EXPECT_EQ((*msg)[am::strings::msg_params][am::strings::result_code].asInt(),
+              (code));
     EXPECT_EQ((*msg)[am::strings::msg_params][am::strings::info].asString(),
               info);
   }
 
+  void SetupIconParameter(MessageSharedPtr msg,
+                          const std::string& type,
+                          const std::string& value) {
+    smart_objects::SmartObject icon =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+    icon[am::strings::type] = type;
+    icon[am::strings::value] = value;
+    (*msg)[am::strings::msg_params][am::strings::audio_pass_thru_icon] = icon;
+  }
+
+  void DefineUIsAvailable() {
+    ON_CALL(hmi_interfaces_,
+            GetInterfaceState(am::HmiInterfaces::HMI_INTERFACE_UI))
+        .WillByDefault(Return(am::HmiInterfaces::STATE_AVAILABLE));
+  }
+
+  void DefineTTSsAvailable() {
```

Comment:
@tkireva Maybe need to combine methods DefineUIsAvailable DefineTTSsAvailable to one method DefineInterfacesAvailable(interface);